### ISSUE TITLE
Spring CFD fails on atlas

### ISSUE THREAD

**Issue by francoisdh** [feelpp/toolbox#68]
This case fails using 

`feelpp_toolbox_fluid --case "github:{repo:toolbox, branch:master, path:examples/modules/cfd/examples/spring_cfd}"`

The mesh is missing (it should be downloaded from girder).

**Comment by celineCQ** [comment]
Did the fix from @francoisdh fix the issue ?

**Comment by vincentchabannes** [comment]
@francoisdh have you remove the scaling in the mesh generation? is it works?

**Comment by celineCQ** [comment]
I still can see the scaling.

**Comment by prudhomm** [comment]
@francoisdh where do we stand on this ?

**Comment by francoisdh** [comment]
@prudhomm The example actually runs fine now after the girder fix (tested with Singularity/atlas), but I have not removed the scaling yet. I was stuck for a while because I was not aware the github path had slightly changed: I was thus using the wrong command and faced a quite misleading error message, which mentionned json parsing error and suggested checking it with yamllint and jsonlint. And so I was checking and checking again the json file... which was fine actually. 

@vincentchabannes I'm working on removing the scaling, but I have a problem at the remeshing step (in AngioTK, see issue feelpp/angiotk#39 and feelpp/angiotk@f65f5606168f80ff925d033c36268454f5b65428). Would you have an idea ?

**Comment by prudhomm** [comment]
@francoisdh what is "for a while" ? you can always run without the case _github_ feature. Regarding the message feel free to fill an issue

**Comment by francoisdh** [comment]
Here, "for a while"  means "in the mean time": between the girder fix and tuesday. After I posted the girder fix, I was of course able to test without the case github feature, that's how I tested the girder fix in the first place. But I wanted to check if everything was working as expected after the merge. And that's why this issue remained opened while it could have been closed. Sorry for that. The path change was subtle and I didn't expect it, so I simply overlooked it and kept trying with an erroneous command. Although there still is the previously mentioned scaling issue, this one can be closed since the case is actually running fine.
